experimenter/experimenter/settings.py

```diff
@@ -203,6 +203,11 @@
     "nimbus-experiment-rest-v6-draft-detail",
     "nimbus-experiment-rest-v7-list",
     "nimbus-experiment-rest-v7-detail",
+    "nimbus-experiment-rest-v8-list",
+    "nimbus-experiment-rest-v8-detail",
+    "nimbus-experiment-rest-v8-first-run-list",
+    "nimbus-experiment-rest-v8-draft-list",
+    "nimbus-experiment-rest-v8-draft-detail",
```

Comment:
We can probably scrap these in V8 since these are for the mobile first run flow, which jetstream shouldn't care about @mikewilli ?